proxy.go

```diff
@@ -87,7 +88,10 @@ func (p *Proxy) notAuthorized(w http.ResponseWriter, r *http.Request) {
 }
 
 func (p *Proxy) allowedResponse(w http.ResponseWriter, r *http.Request) {
-	if err := p.auth.Authenticate(r); err != nil {
+	var claims jwt.MapClaims
+	var err error
+
+	if claims, err = p.auth.Authenticate(r); err != nil {
```

Comment:
```suggestion
	claims, err = p.auth.Authenticate(r)
	if err != nil {
```
Everything is within the same function call so keep i as simple as possible.

Reply:
This is an issue in general in this repo. We have many one-liners that could (and probably should) be broken down. Maybe we add this to the refactoring sprint?

Reply:
This can be fixed now, other things like converting the nested if blocks to switch statements is something we can do as a refactoring